model/kubernetes/kubernetes_cluster.rb

```diff
@@ -0,0 +1,65 @@
+#  frozen_string_literal: true
+
+require_relative "../../model"
+
+class KubernetesCluster < Sequel::Model
+  one_to_one :strand, key: :id
+  one_to_one :api_server_lb, class: :LoadBalancer, key: :id, primary_key: :api_server_lb_id
+  many_to_one :private_subnet
+  many_to_one :project
+  many_to_many :cp_vms, join_table: :kubernetes_clusters_cp_vms, class: :Vm, order: :created_at
+
+  dataset_module Pagination
+
+  include ResourceMethods
+  include SemaphoreMethods
+  include Authorization::HyperTagMethods
```

Comment:
Jeremy's PR will likely be merged before this one. We should get rid of `Authorization::HyperTagMethods` and `hyper_tag_name`.

https://github.com/ubicloud/ubicloud/pull/2616

Reply:
Merged

Reply:
removeed